core/plugins/org.polarsys.capella.core.ui.search/src/org/polarsys/capella/core/ui/search/CapellaSearchResultLabelProvider.java

```diff
@@ -55,27 +64,28 @@ public StyledString getStyledText(Object element) {
     StyledString str = new StyledString();
 
     str.append(getLabelText(element));
-    str.append(" "); 
+    str.append(" ");
 
     if (element instanceof EObject) {
       boolean isLockedByOther = CapellaReadOnlyHelper.getReadOnlySectionHandler().isLockedByOthers((EObject) element);
       if (isLockedByOther) {
-        str.append("(Read-Only)", StyledString.DECORATIONS_STYLER); 
+        str.append("(Read-Only)", StyledString.DECORATIONS_STYLER);

```

Comment:
For that, i think we can use PlatformUI.getDecoratorManagerXX to use decorations used in Preferecnes / General / Appareance / Label Decorations.

Reply:
This is a great idea. Now we have all the lock decorations under T4C :)

Reply:
There is something here:
tps://github.com/eclipse/capella/blob/b05bc20ce8fc38b98ac7789c16c0f347d10d050b/core/plugins/org.polarsys.capella.core.ui.toolkit/src/org/polarsys/capella/core/ui/toolkit/dialogs/ConfirmDeleteCapellaElementDialog.java#L123